Event.js

```diff
@@ -0,0 +1,132 @@
+import * as L from "./Logger.js";
+import * as Validate from "./Validators.js";
+
+class Event {
+    #state;
+
+    constructor(state) {
+        this.#state = state;
+    }
+
+    /**
+     * Used to send various types of event;
+     *
+     * @param {string} eventName event name.
+     * @param {number} eventCount event count.
+     * @param {number} eventSum event sum.
+     * @param {Segmentation} segments event segmentation.
+     * @return {string | void} error message or void
+     */
+    recordEvent(eventName, eventCount, eventSum, segments) {
+        if (!this.#state.isInitialized) {
+            const message = "'init' must be called before 'recordEvent'";
+            L.e(`recordEvent, ${message}`);
+            return message;
```

Comment:
@ArtursKadikis shall we stop returning messages under the new interface?

Reply:
yup, in fact, we do want to do that :D

Reply:
done